src/lib/include/simd_config.h

```diff
@@ -57,4 +57,11 @@ static_assert(!DISTOPIA_USE_AVX2 || !DISTOPIA_USE_AVX,
 static_assert(!DISTOPIA_USE_AVX512 || !DISTOPIA_USE_AVX2,
               "SIMD config is not self-consistent");
 
+// now include headers for simd wrappers
+// these headers contain logic to include correct methods based on defines above
+//#include "simd/simd_double_datastructures.hpp"
+#include "simd/simd_doubles.h"
+#include "simd/simd_float_datastructures.hpp"
+#include "simd/simd_floats.h"
```

Comment:
This is a very low-level file. I wouldn’t include a bunch of high-level data structures here.